.gitignore

```diff
@@ -1,2 +1,3 @@
 Cargo.lock
-target
\ No newline at end of file
+target
+.vscode
```

Comment:
FWIW, you can make a `~/.gitignore` and put your editor specific stuff there, then you won't need to worry about adding these things to individual repos. 

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
sounds good, done.